.github/workflows/run-archiver.yml

```diff
@@ -18,9 +18,11 @@ jobs:
           - eia191
           - eia757a
           - eia860
+          - eia861
           - eia860m
           - eia923
           - eia_bulk_elec
+          - phmsagas
```

Comment:
I notice that this job is called `archive-run-eia` but it contains non-EIA datasets. Do we intend for the jobs to be broken out by data source? It seems like the workflow is primarily organized by whether a small vs. large runner is required.

It's not clear to me whether we really need to fix the cause of the "previous version" errors for `mshamines` and `epacamd_eia` that you note in #285 before adding those datasets in here (or ever). If approving the draft archives will make future versions work, I think it's just a one-time change in the available metadata and shouldn't be a problem in the future should it? (or rather, it'll be a problem, but just once for each dataset, with the same solution, of approving the new draft archive despite the error).

Reply:
Great catch. I'll rename this job and run both to see if manual approval fixes the error, then add them back.